src/meta/geo/geoStatistics.ts

```diff
@@ -9,10 +9,17 @@ export type ExtraEstimationSectionState = Record<ExtraEstimation, ExtraEstimatio
 
 export type GeoStatisticsExtraEstimations = Record<LayerSectionKey, ExtraEstimationSectionState>
 
+export type ForestEstimationEntry = {
```

Comment:
This type change is part of a bigger refactoring that will be completed after finishing the GEO style changes.